client/stylesheets/_toCat.scss

```diff
@@ -4794,7 +4794,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .searchResultsCardPrice {
-    margin: 0 0 15px 0;
+    margin: 10px 0 15px 0;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `10px 0 15px` instead of `10px 0 15px 0`